src/openrct2/world/MapAnimation.h

```diff
@@ -43,3 +45,4 @@ void MapAnimationCreate(int32_t type, const CoordsXYZ& loc);
 void MapAnimationInvalidateAll();
 const std::vector<MapAnimation>& GetMapAnimations();
 void MapAnimationAutoCreate();
+void MapAnimationAutoCreateTileElement(TileCoordsXY coords, TileElement* el);
```

Comment:
I feel the name is slightly odd it suggests its creating a tile element. Maybe stick a `For` in there. `MapAnimationAutoCreateForTileElement`

Reply:
I've just changed the name to `MapAnimationAutoCreateAtTileElement`